news/4553.feature

```diff
@@ -0,0 +1 @@
+pip now displays a warning when it installs scripts outside PATH. These warnings can be suppressed using a new --no-warn-script-location option.
```

Comment:
Maybe also mention, that it will only work with wheel installs (or sdist automatically converted to wheel)